cluster-provision/k8s/1.16/sonobuoy_version

```diff
@@ -0,0 +1 @@
+0.18.2
```

Comment:
conformance version is inside `conformance.json` we don't need new file, `jq .Version` will give it to us.

Reply:
Not sure whether we have jq installed in test image. Let me check.

Reply:
we usually use a containerized version of it if needed, Quique added it once